Rubberduck.Refactorings/MoveCloserToUsage/MoveCloserToUsageModel.cs

```diff
@@ -1,14 +1,27 @@
-﻿using Rubberduck.Parsing.Symbols;
+﻿using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
 
 namespace Rubberduck.Refactorings.MoveCloserToUsage
 {
     public class MoveCloserToUsageModel : IRefactoringModel
     {
-        public Declaration Target { get; }
+        private Declaration _target;
+        public Declaration Target
+        {
+            get => _target;
+            set
+            {
+                _target = value;
+            }
+        }
+
+
+        public string DeclarationStatement { get; set; } = string.Empty;
 
         public MoveCloserToUsageModel(Declaration target)
         {
             Target = target;
         }
+
```

Comment:
Applied this in [c05a29c](https://github.com/rubberduck-vba/Rubberduck/pull/5941/commits/c05a29c2fa0de5ca50876dbede88055170dec760)

I changed the Parameter/Property to `VariableDeclaration`. Maybe you take a look at this because I'm not sure how to implement the optional parameter best (constructor chaining and overwrite the auto property again?)

Reply:
No need to chain.  I was thinking a single constructor something like...
```
public MoveCloserToUsageModel(VariableDeclaration target, string declarationStatement = null)
{
          _target = target;
         DeclarationStatement = declarationStatement ?? string.Empty;
}
```

Reply:
Thank you, I was thinking too complicated. Now I changed this as you suggested: [be40f30](https://github.com/rubberduck-vba/Rubberduck/pull/5941/commits/be40f305e2ed78249b470bfc04dc94d64e1865c8)